onboarding/README.md

```diff
@@ -0,0 +1,60 @@
+# 10101 Onboarding
+
+Our 10101.finance routing node acts as LSP and will create a just-in-time channel with you once the first payment is received.
+This channel will have a capacity of 200k sats.
+Currently don't take any fees for creating that channel, but we have a fee policy of 2% and 1000 msats base fee to cover some of the costs for opening these channels.
+So make sure that your fee limits cover for that.
+
+Please also note, that we are not yet well connected to the network, so it may take a few hops before you will be able to reach us.
+
+The following sections provide some guidance on how to fund the 10101 wallet from the different apps.
+
+## Onboarding through a mobile app
+
+If you want to fund 10101 through another mobile app on your phone, make sure that you have started 10101 in the background and switch back to it after sending the payment from the onboarding wallet.
+The 10101.finance LSP will wait 30 seconds for the 10101 app to come back online before the payment is marked as failed.
+
+## Onboarding through the Phoenix Wallet
+
+You need to increase your maximum fee for outgoing Lightning payments.
+Follow these steps to get you going.
+
+1. Open your Phoenix Wallet on your phone.
+2. Go To Settings / Payment options & fees
+3. Click on Maximum fee for outgoing Lightning payments and set it to 20 sats + 2,00%
```

Comment:
wouldn't these instructions be soon deprecated in light of https://github.com/get10101/10101/pull/579 ? (hopefully resulting in much less onboarding troubles)